packages/mux-uploader/package.json

```diff
@@ -46,7 +46,7 @@
     "publish-release": "../../scripts/publish.sh"
   },
   "dependencies": {
-    "@mux/upchunk": "^2.3.1"
+    "@mux/upchunk": "^2.5.0"
```

Comment:
todo(blocker): This should be upgraded to 2.6.0, since there was another release cut.

Reply:
✅